src/mgmt/cli_mgmt.c

```diff
@@ -441,6 +441,90 @@ int dc_mgmt_net_cfg(const char *name)
 	return rc;
 }
 
+static int issue_monitor_request(struct dc_pool *pool, int request_type)
+{
+	struct drpc_alloc	 alloc = PROTO_ALLOCATOR_INIT(alloc);
+	struct drpc		 *ctx;
+	Mgmt__PoolMonitorReq	 req = MGMT__POOL_MONITOR_REQ__INIT;
+	uint8_t			 *reqb;
+	size_t			 reqb_size;
+	char			 pool_uuid[37], pool_hdl_uuid[37];
```

Comment:
We have a DAOS_UUID_STR_SIZE defined somewhere, should use that.

Reply:
Fixed